apps/studio/src/server/modules/resource/resource.router.ts

```diff
@@ -35,7 +38,7 @@ const fetchResource = async (resourceId: string | null) => {
   )
 }
 
-const validateUserPermissions = async ({
+const validateUserPermissionsForMove = async ({
```

Comment:
nit: actually since you already moved `validateUserPermissions` into the permissions.service file, I think this function should be moved there as well so we can consolidate all functions related to permissions validation into a single place.